bin/wv-options-build

```diff
@@ -22,26 +22,29 @@ die() {
     exit 1
 }
 
-ENV=$1
-
 [ -d "$SRC_DIR" ] || die "Options directory does not exist"
 
+# Activate virtual Python environment
+PATH=.node-virtualenv/bin:.node-virtualenv/Scripts:${PATH}
+
+# If $FETCH_GC is set fetch GetCapabilities
 if [ "$FETCH_GC" ] ; then
-    rm -rf "$OPT_DIR/$ENV/gc/*"
-    rm -rf "$OPT_DIR/$ENV/colormaps/gc/*"
-    "$BIN_DIR/wv-options-fetch" "$OPT_DIR/$ENV/config.json" "$OPT_DIR/$ENV/gc"
+    rm -rf "$OPT_DIR/release/gc/*"
```

Comment:
As far as I can tell, this is the only value that `$ENV` ever has, so I removed the variable to simplify the command that needs to be called to build the options repo.

Reply:
@localjo the `v$ENV` is used by gibs. They have many different vars. An example would be `sit-ext`.